yt/visualization/color_maps.py

```diff
@@ -48,26 +52,19 @@ def register_yt_colormaps_from_cmyt():
     """
 
     for hist_name, alias in _HISTORICAL_ALIASES.items():
-        cmap = mpl.colormaps[alias].copy()
-        cmap.name = hist_name
-        mpl.colormaps.register(cmap)
-        mpl.colormaps.register(cmap.reversed())
+        cmap = mpl.colormaps[alias]
+        cmap_r = mpl.colormaps[f"{alias}_r"]
+
+        mpl.colormaps.register(cmap, name=hist_name)
+        mpl.colormaps.register(cmap_r, name=f"{hist_name}_r")
 
 
 register_yt_colormaps_from_cmyt()
 
 # Add colormaps in _colormap_data.py that weren't defined here
-_vs = np.linspace(0, 1, 256)
-for k, v in list(_cm.color_map_luts.items()):
-    if k in yt_colormaps:
-        continue
-    cdict = {
-        "red": np.transpose([_vs, v[0], v[0]]),
-        "green": np.transpose([_vs, v[1], v[1]]),
-        "blue": np.transpose([_vs, v[2], v[2]]),
-    }
+for k, v in _cm.color_map_luts.items():
     try:
-        add_colormap(k, cdict)
+        mpl.colormaps.register(ListedColormap(v, name=k, N=256))
```

Comment:
I mean, this is slightly flatter, but I don't think it's a big deal, either.

Reply:
yeah, this PR yields disappointing results, but just in case, I want to clarify that the gain here is not expected to come from the *inlining*, but from instantiating `ListedColormap` instead of `LinearSegmentedColormap`, which saves a conversion from the format we actually define our maps in.

Reply:
any way #4538 and #4517 are both much more efficient at reducing the startup time, I really don't mind if we end up just closing this one.